rust/operator/src/kafka_controller.rs

```diff
@@ -767,26 +763,30 @@ fn build_broker_rolegroup_statefulset(
         .add_volume_mount(LOG_DIRS_VOLUME_NAME, STACKABLE_DATA_DIR)
         .add_volume_mount("config", STACKABLE_CONFIG_DIR)
         .add_volume_mount("tmp", STACKABLE_TMP_DIR)
-        .resources(resources);
-
-    // Use kcat sidecar for probing container status rather than the official Kafka tools, since they incur a lot of
-    // unacceptable perf overhead
-    let mut container_kcat_prober = cb_kcat_prober
-        .image("edenhill/kcat:1.7.0")
-        .command(vec!["sh".to_string()])
-        // Only allow the global load balancing service to send traffic to pods that are members of the quorum
-        // This also acts as a hint to the StatefulSet controller to wait for each pod to enter quorum before taking down the next
+        .startup_probe(Probe {
+            tcp_socket: Some(TCPSocketAction {
+                port: IntOrString::Int(kafka.client_port().into()),
+                ..TCPSocketAction::default()
+            }),
+            failure_threshold: Some(10),
```

Comment:
20s to start up seems a little bit short

Reply:
yeah i want to tune these numbers today a little bit more